examples/resources/temporalcloud_user/resource.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  required_providers {
+    temporalcloud = {
+      source = "temporalio/temporalcloud"
+    }
+  }
+}
+
+provider "temporalcloud" {
+
+}
+
+resource "temporalcloud_namespace" "namespace" {
+  name               = "terraform"
+  regions            = ["aws-us-east-1"]
+  accepted_client_ca = base64encode(file("${path.module}/ca.pem"))
+  retention_days     = 14
+}
+
+resource "temporalcloud_user" "global_admin" {
+  email          = "admin@yourdomain.com"
+  account_access = "admin"
+}
+
+resource "temporalcloud_user" "namespace_admin" {
```

Comment:
if you drop a bash script named `import.sh` in this directory, the documentation generator will generate a little stanza in the docs about how to import the resource, like this: https://github.com/temporalio/terraform-provider-temporalcloud/blob/main/examples/resources/temporalcloud_namespace/import.sh